recipes/cargo-llvm-cov/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.6.9" %}
+
+package:
+  name: cargo-llvm-cov
+  version: {{ version }}
+
+source:
+  url: https://github.com/taiki-e/cargo-llvm-cov/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 0a1b63922cfb6f2526a61bc5e13868c3f3345aa3af7337df07a69eab3d4b243e
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}  # [unix]
+    - {{ stdlib('c') }}    # [unix]
+    - cargo-bundle-licenses
+  host:
+    - clangdev  # [not osx]
+    - llvmdev   # [not osx]
```

Comment:
I would suspect that we also need them on `osx` but simply link against the compilers that are installed in `build`. Also, do you know which parts of `llvm` are used?

Reply:
Seems like these host dependencies are, in fact, not required. I originally copied this recipe from [taplo](https://github.com/conda-forge/taplo-feedstock) to get started and I was happy once it built, hence, didn't try to find the minimal set of build dependencies :eyes: